lib/puppet/module_tool/applications/builder.rb

```diff
@@ -40,6 +41,22 @@ def archive_file
         File.join(@pkg_path, "#{metadata.release_name}.tar.gz")
       end
 
+      def minitar_verify
+        begin
+          # assume valid if minitar inaccessible (i.e. immediate rescue loading gem)
+          require 'archive/tar/minitar'
```

Comment:
The general idea here is that modules would opt into this behavior by adding `minitar` to their `Gemfile`.

This wouldn't take effect under normal circumstances.